app/folder/layout.tsx

```diff
@@ -0,0 +1,11 @@
+import BaseLayout from "@/components/layout/BaseLayout";
+import React from "react";
+
+type Props = {
+  children: React.ReactNode;
+};
```

Comment:
React.ReactNode와 같이 사용하는 경우에는 `import ... from "react"`와 같이 react라는 패키지에서 무언가를 가져와서 사용하는 경우에도 일관되게 React를 붙여주는게 좋다고 생각하는데요!
예를 들면, `React.useState`,  `React.useEffect`, `React.useCallback` 이런 식으로요!
그런데 다른 파일을 보면 위와 같이 작성되지 않았기 때문에 여기서는 `React.`없이 `ReactNode`로 바로 표현해줘도 좋을 것 같아요!

```suggestion
type Props = {
  children: ReactNode;
};
```